### ISSUE TITLE
Any interest in updates?

### ISSUE THREAD

**Issue by hyegeek** [qwertangel/Hello-You-There#1]
While this project is pretty old, the audio files are perfect for trolling the scammers. After setting up the dial plan on my pbx, I wanted to be sure I understood what it was doing. The result of that was a re-write to simplify things a bit and to adjust the silences to flow more naturally.

I'd be happy to contribute what I've done if there is any interest.

**Comment by carefulcomputer** [comment]
absolutely. would be great to have your contribution. would you be sending a pull request ?

**Comment by hyegeek** [comment]
I'm new to github, so while that is what I want to do, I'm needing to figure out how. :-)

**Comment by hyegeek** [comment]
I've committed my changed to my local repository and tried to create a pull request, but the button is grayed out. I'm guessing I don't have permission. Or I need to create a branch or something like that.

**Comment by carefulcomputer** [comment]
try this..

https://help.github.com/articles/editing-files-in-another-user-s-repository/
Editing files in another user's repository - User ...<https://help.github.com/articles/editing-files-in-another-user-s-repository/>
help.github.com
In another user's repository, browse to the folder that contains the file you want to edit. Click the name of the file you want to edit. Above the file content, click .



________________________________

**Comment by hyegeek** [comment]
Thanks. I think I did it. At least the main pages mentions my patch. Let me know what you think.

**Comment by carefulcomputer** [comment]
I can see it now. I created a Pull request so that I can merge it in main repo. Script does look more organized now ! Will merge after I run through it once.


Thanks !

________________________________

**Comment by hyegeek** [comment]
It just occurred to me that I'm pointing to a different directory for the sound files. I ended up making a link when I was testing, but I wanted to be able to mess with my own copy without breaking the old one.

**Comment by carefulcomputer** [comment]
updated the folders to match script. Please check.

**Comment by hyegeek** [comment]
Looks good to me. While going over the code again last night, I noticed room for another minor improvement. I've submitted that patch as well.

**Comment by carefulcomputer** [comment]
i see that mixmonitor was removed and another line was moved out of loop. is that what you wanted to do ?

________________________________

**Comment by hyegeek** [comment]
Serves me right for doing things quickly before I leave. That patch did not look like what I wanted. Let me do another that is correct....

OK. New patch looks like what I really wanted. MixMonitor moved.

**Comment by carefulcomputer** [comment]
I see there are several patches  you created. Some are related to lenny. Which patch do you think I should pickup ?

________________________________

**Comment by hyegeek** [comment]
One of them is for helloyouthere.txt, that's the one that fixes what I mentioned before.

There are 3 other patches that all add files. These are for a proposed way to allow for easy expansion to other sets of clips. The first teletrap.conf would be an include file that would do the work of playing the clips. The other two files are examples of using the teletrap.conf file. One implements the same things that helloyouthere.txt does, the other is an example of implementing lenny with the same include.

If you think this ia a good way to go, you can accept the files, if not, just ignore them and only updated the helloyouthere.txt file.

**Comment by hyegeek** [comment]
Looking at the list of patches, patch-3 is the one that fixes helloyouthere.txt. The other files patch-4-6, are the patches that add new files.

**Comment by carefulcomputer** [comment]
just merged patch 3. thanks for contribution ! I am sure this script will help lot of people.

________________________________
